device/config/configurator.go

```diff
@@ -110,28 +103,296 @@ func (c *configurator) Write() error {
+	if err != nil {
+		return errors.New(err.Error() + ":" + eut)
+	}
+
+	return nil
+}
+
+// SetInterface is a dialog asking to setup user Interfaces for the static ip functionality
+func SetInterface(storage map[string]interface{}) error {
+	log.WithField("type", "default").Debug("setInterface")
+	device := []string{"eth0", "wlan0"}
+	i := Interfaces{
+		Address: "192.168.0.254",
+		Netmask: "255.255.255.0",
+		Gateway: "192.168.0.1",
+		DNS:     "192.168.0.1",
+	}
+
+	if dialogs.YesNoDialog("Would you like to assign static IP address for your device?") {
+		fmt.Println("[+] Available network interface: ")
+		num := dialogs.SelectOneDialog("Please select a network interface: ", device)
+		fmt.Println("[+] ********NOTE: ADJUST THESE VALUES ACCORDING TO YOUR LOCAL NETWORK CONFIGURATION********")
+
+		fmt.Printf("[+] Current values are:\n \t[+] Address:%s\n\t[+] Gateway:%s\n\t[+] Netmask:%s\n\t[+] DNS:%s\n",
+			i.Address, i.Gateway, i.Netmask, i.DNS)
+
+		if dialogs.YesNoDialog("Change values?") {
+			AskInterfaceParams(&i)
+		}
+
+		switch device[num] {
+		case "eth0":
+			storage[Interface] = fmt.Sprintf(InterfaceETH, i.Address, i.Netmask, i.Gateway, i.DNS)
+			fmt.Println("[+]  Ethernet interface configuration was updated")
+		case "wlan0":
+			storage[Interface] = fmt.Sprintf(InterfaceWLAN, i.Address, i.Netmask, i.Gateway, i.DNS)
+			fmt.Println("[+]  wifi interface configuration was updated")
+		}
+
+	}
+
+	return nil
+}
+
+// SaveInterface is a default method to save user Interfaces into the image
+func SaveInterface(storage map[string]interface{}) error {
+
+	if _, ok := storage[Interface]; !ok {
+		return nil
+	}
+
+	ssh, ok := storage["ssh"].(ssh_helper.Util)
+	if !ok {
+		return errors.New("Cannot get ssh config")
+	}
+
+	fp := help.AddPathSuffix("unix", MountDir, ISAAX_CONF_DIR, "network", "interfaces")
+
+	_, eut, err := ssh.Run(fmt.Sprintf(`echo "%s" > %s`, storage[Interface], fp))
+	if err != nil {
+		return errors.New(err.Error() + ":" + eut)
+	}
+
+	return nil
+}
+
+// SetSecondaryDNS is a dialog asking to set 8.8.8.8 DNS
+func SetSecondaryDNS(storage map[string]interface{}) error {
+	if dialogs.YesNoDialog("Add Google DNS as a secondary NameServer") {
+		storage[DNS] = true
+		return nil
+	}
+
+	return nil
+}
+
+// SaveSecondaryDNS is a default method to set 8.8.8.8 as a secondary DNS
+func SaveSecondaryDNS(storage map[string]interface{}) error {
+
+	if _, ok := storage[DNS]; !ok {
+		return nil
+	}
+
+	ssh, ok := storage["ssh"].(ssh_helper.Util)
+	if !ok {
+		return errors.New("Cannot get ssh config")
+	}
+
+	fp := help.AddPathSuffix("unix", MountDir, ISAAX_CONF_DIR, "dhcp", "dhclient.conf")
+	command := "append domain-name-servers 8.8.8.8, 8.8.4.4;"
+
+	_, eut, err := ssh.Run(fmt.Sprintf(`echo "%s" >> %s`, command, fp))
+	if err != nil {
+		return errors.New(err.Error() + ":" + eut)
+	}
+
+	return nil
+}
+
+// SetInterfaces is a set of dialog to set user `Interfaces`
```

Comment:
__[golint]__
* [golint] comment on exported function AskInterfaceParams should be of the form "AskInterfaceParams ..."

<!-- Comment by [SideCI](https://sideci.com) -->